### ISSUE TITLE
CPU consumption - Feature request: configurable polling time

### ISSUE THREAD

**Issue by ramack** [vikt0rm/dbus-shelly-1pm-pvinverter#22]
May I propose to make the polling time a configuration option?

I recognized a high CPU load (around 15-20%) by the shelly-1pm driver which is really an issue in my beaglebone base VenusOS, taking care of also 4 BMSs with the serial-driver.

It [looks like](https://github.com/vikt0rm/dbus-shelly-1pm-pvinverter/blob/307813021a2b6eb5370ec96af263bd4e7e4ec625/dbus-shelly-1pm-pvinverter.py#L64) the driver is polling 4 times per second and in my setup the shelly data is really low priority compared to the batteries, which I poll only every 5 seconds to keep the system up, so I'd appreciate if I could reduce the CPU load by increasing the poll interval for the shelly.

As a default value I would even propose to go for 1 second...

**Comment by emuman049** [comment]
Makes sense. For now, you could increase the value in line 64 to 1000 or more.
